### PR TITLE
Add Quobyte's CSI plug-in reference in place of in-tree plug-in

### PR DESCRIPTION
**Pull request by venkatsc**
Added new reference link for Quobyte CSI in place of old in-tree plug-in.
Quobyte supports CSI plug-in and is the recommended plug-in to access the volumes.

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -835,7 +835,9 @@ You must have your own Quobyte setup running with the volumes
 created before you can use it.
 {{< /caution >}}
 
-See the [Quobyte example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/quobyte) for more details.
+Quobyte supports [Container Storage Interface](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/).
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md]:
```suggestion
Quobyte supports the {{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}}.
```
I'm not sure whether the website style guide prefers CSI to Container Storage Interface there, but it doesn't read as well anyway. The [glossary entry](https://kubernetes.io/docs/reference/glossary/?all=true#term-csi) is a good place to link to and will have up-to-date signposting to further reading etc.

File: content/en/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -835,7 +835,9 @@ You must have your own Quobyte setup running with the volumes
...
+CSI is the recommended plugin to use Quobyte volumes inside Kubernetes. Quobyte's
+CSI deployment instructions and examples can be found [here](https://github.com/quobyte/quobyte-csi).
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md]:
```suggestion
GitHub project has [instructions](https://github.com/quobyte/quobyte-csi#quobyte-csi) for deploying Quobyte using CSI, along with examples.
```

**Comment by zacharysarah** [comment]
@venkatsc Thanks for the update. Welcome to Kubernetes! 🎉 

/lgtm
/approve

**Comment by venkatsc** [comment]
Thank you @zacharysarah @sftim for reviewing the change.

**Pull request merged**